### PR TITLE
Bug Fix 

### PR DESCRIPTION
**Pull request by tahmid-77**
Width issues and extra classes are removed.

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -122,25 +117,23 @@
...
+
+.feature-image {
+  margin: 2px;
 }
 
-.feature-image-1 {
-  width: 310px;
-  height: 428px;
-  padding: 100px 75px 0px;
+.flower-image-1 {
```
Comment (yafi38) [inline index.css:127]: Please remove the double semicolons

File: index.css [hunk 2]
```diff
@@ -122,25 +117,23 @@
...
+.flower-image-2 {
+  padding: 120px 75px 40px 50px;;
 }
 
-.feature-image-3 {
-  width: 310px;
-  height: 470px;
-  padding: 58px 75px 0px;
+.flower-image-3 {
+  padding: 54px 75px 0px;;
```
Comment (yafi38) [inline index.css:136]: Please remove the double semicolons

**Review by yafi38 — CHANGES_REQUESTED** [review]

File: index.css (hunk 2 above)
Comment (tahmid-77) [reply index.css]: Removed.

**Review by yafi38 — APPROVED** [review]
Looks good to me 👍

**Pull request merged**